api/src/main/java/com/basistech/rosette/api/RosetteAPI.java

```diff
@@ -137,9 +137,11 @@ public RosetteAPI(String key) throws IOException, RosetteAPIException {
      * @throws IOException         General IO exception
      */
     public RosetteAPI(String key, String alternateUrl) throws IOException, RosetteAPIException {
+        boolean shouldCheckVersion = false;
         urlBase = alternateUrl;
         if (urlBase == null) {
-            checkVersionCompatibility();
+            urlBase = DEFAULT_URL_BASE;
```

Comment:
You're right, this isn't the correct way to fix the problem. I'll amend in a second with what should be a better way of handling the constructors.

My concern was using the constructor that just takes an API key. It passes null to the constructor that takes both API key and alternateUrl. I interpreted the existing code as wanting to check version compatibility when an alternateUrl isn't specified. Given that you need a urlBase to check the compatibility, it makes sense to give urlBase the default.